### PR TITLE
Remove inline styles from Web/API docs

### PR DESCRIPTION
**Pull request by Blakelist7**
Part of fixing #5438

### CODE REVIEW

File: files/en-us/web/api/pushevent/index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@ <h2 id="Properties">Properties</h2>
  <dd>Returns a reference to a {{domxref("PushMessageData")}} object containing data sent to the {{domxref("PushSubscription")}}.</dd>
 </dl>
 
-<p><strong style="font-size: 2.14285714285714rem; font-weight: 700; letter-spacing: -1px; line-height: 30px;">Methods</strong></p>
+<p><strong>Methods</strong></p>
```
Comment (hamishwillee) [inline files/en-us/web/api/pushevent/index.html:36]: Should this actually be a heading style?

File: files/en-us/web/api/request/request/index.html [hunk 2]
```diff
@@ -163,8 +163,8 @@ <h2 id="Example">Example</h2>
   constructor to create a copy of the Request (This is similar to calling the
   {{domxref("Request.clone","clone()")}} method.)</p>
 
-<div class="codehilite" style="background: #f0f3f3;">
-  <pre style="line-height: 125%;">var copy = new Request(myRequest);
+<div class="codehilite">
```
Comment (hamishwillee) [inline files/en-us/web/api/request/request/index.html:166]: Remove the div and this class, then make the `pre` to use brush for js

**Review by hamishwillee — CHANGES_REQUESTED** [review]
Just 2 little nits. Thanks!

File: files/en-us/web/api/pushevent/index.html (hunk 1 above)
Comment (Blakelist7) [reply files/en-us/web/api/pushevent/index.html:36]: Yes, I missed it 😅

File: files/en-us/web/api/request/request/index.html (hunk 2 above)
Comment (Blakelist7) [reply files/en-us/web/api/request/request/index.html]: Sure

File: files/en-us/web/api/pushevent/index.html [hunk 3]
```diff
@@ -33,7 +33,7 @@ <h2 id="Properties">Properties</h2>
  <dd>Returns a reference to a {{domxref("PushMessageData")}} object containing data sent to the {{domxref("PushSubscription")}}.</dd>
 </dl>
 
-<p><strong>Methods</strong></p>
+<h2><strong>Methods</strong></h2>
```
Comment (hamishwillee) [inline files/en-us/web/api/pushevent/index.html:36]: Please add id="methods" and remove the strong tag :-)

**Review by hamishwillee — APPROVED** [review]
Thanks. Now just one small nit ;-)

File: files/en-us/web/api/pushevent/index.html (hunk 3 above)
Comment (Blakelist7) [reply files/en-us/web/api/pushevent/index.html]: Done

**Review by hamishwillee — APPROVED** [review]
Thanks so much for this!!!!

**Pull request merged**